school_library_decorator/book.rb

```diff
@@ -10,4 +10,23 @@ def initialize(title, author)
   def add_rental(date, person)
     Rental.new(self, date, person)
   end
+
+  def self.create_book
+    print 'title: '
+    title = gets.chomp
+
+    print 'author: '
+    author = gets.chomp
+
+    puts 'Book has been created successfully'
+    Book.new(title, author)
+  end
+
+  def self.all_books(books)
+    if books.length.positive?
+      books.each { |book| puts "title: #{book.title}, Author: #{book.author}" }
+    else
+      puts 'Sorry! Cannot find any book.'
+    end
+  end
```

Comment:
- [ ] Kindly refactor this piece of code to follow the Single responsibility rule, in ruby we can also do more by writing less and by doing we also follow this KISS rule